Libs/MRML/Widgets/qMRMLSliceControllerWidget.h

```diff
@@ -167,6 +169,8 @@ public slots:
   /// \sa setSliceView();
   void fitSliceToBackground();
 
+  void MaximizeLayout();
```

Comment:
method names on classes inheriting from Qt classes should start with lower case letters.